docs/pages/accounts/sso.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Single Sign-On (SSO)
+description: Learn about how your enterprise organization can use your Identity Provider to manage Expo users.
+---
+
+## Configure SSO for your organization
+
+SSO is available for Enterprise Plan customers. To get started, contact us and we'll help you set it up for your organization.
+
+> The Expo SSO system should support any Identity Provider (IdP) that supports [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html). We have tested [Okta](https://www.okta.com/) and plan to test others as requested.
+
+## SSO user sign in
+
+### Expo website
+
+1. Navigate to https://expo.dev/sso-login and enter the account name of your organization.
+
+    > You can create a link that pre-fills the organization name. For example, https://expo.dev/sso-login/test-org will pre-fill `test-org`.
+
+1. Log in to your identity provider (IdP).
+1. You will then be asked to choose a Expo username. This will be your username in Expo.
+
+### Expo CLI
+
+`npx expo login --sso`
+
+### EAS CLI
+
+`eas login --sso`
+
+### Expo Go
+
+1. Go through normal sign in flow but click "Continue with SSO" button on sign in page.
```

Comment:
```suggestion
1. Click the **Continue with SSO** button on the sign-in page when going through the sign-in flow.
```

I think we can put attention to the different step that user has to take when going through this step.